### PR TITLE
send  keys and status  info periodically to panel

### PR DESCRIPTION
**Pull request by JohaoRosasRosillo**

### CODE REVIEW

File: lib/agent/plugins/control-panel/index.js [hunk 1]
```diff
@@ -100,10 +110,39 @@ function handle_response(what, err, resp) {
     commands.process(resp.body);
 }
 
+var compatible_with_module_tpm = function (data) {
```
Comment (patriciojofre) [inline lib/agent/plugins/control-panel/index.js:113]: maybe this function could be defined in a more global context to be available to other parts of the project

File: lib/agent/plugins/control-panel/index.js [hunk 2]
```diff
@@ -100,10 +110,39 @@ function handle_response(what, err, resp) {
...
+}
+
+function send_info_encrypt(params,cb) {
+  let data = {};
+  if(os_name == 'windows'){
+    system.get_os_edition(function(err, os_edition) {
+      if (err) return new Error('Error to get os_edition informaton');
+      data.os_edition = os_edition;
+      system.get_winsvc_version(function(err, winsvc_version) {
+        if (err) return new Error('Error to get winsvc_version informaton');
```
Comment (patriciojofre) [inline lib/agent/plugins/control-panel/index.js:131]:
```suggestion
        if (err) return new Error('Error to get winsvc_version information');
```

File: lib/agent/plugins/control-panel/index.js [hunk 3]
```diff
@@ -100,10 +110,39 @@ function handle_response(what, err, resp) {
...
+      data.winsvc_version && gte(data.winsvc_version, "2.0.0"))
+        return true;
+  return false;
+}
+
+function send_info_encrypt(params,cb) {
+  let data = {};
+  if(os_name == 'windows'){
+    system.get_os_edition(function(err, os_edition) {
+      if (err) return new Error('Error to get os_edition informaton');
```
Comment (patriciojofre) [inline lib/agent/plugins/control-panel/index.js:128]:
```suggestion
      if (err) return new Error('Error to get os_edition information');
```

File: lib/agent/plugins/control-panel/index.js [hunk 4]
```diff
@@ -57,9 +60,16 @@ var wait_for_config = function(cb) {
...
 }
 
+//exports.timeout =  1 * 60 * 1000   // Every 1 minutes
+exports.timeout = 2 * 60 * 60 * 1000;  // Every 2 hours
 function boot(cb) {
   load_hooks();
   sync();
+  
+  var status_interval = setInterval(() => {
+    send_info_encrypt();
```
[the remark this answers is not in the document]
Comment (JohaoRosasRosillo) [reply lib/agent/plugins/control-panel/index.js:70]: done

File: lib/agent/plugins/control-panel/index.js (hunk 1 above)
Comment (JohaoRosasRosillo) [reply lib/agent/plugins/control-panel/index.js]: done

File: lib/agent/plugins/control-panel/index.js (hunk 2 above)
Comment (JohaoRosasRosillo) [reply lib/agent/plugins/control-panel/index.js]: done

File: lib/agent/plugins/control-panel/index.js (hunk 3 above)
Comment (JohaoRosasRosillo) [reply lib/agent/plugins/control-panel/index.js]: done

**Pull request merged**